src/amo/routes.js

```diff
@@ -5,8 +5,7 @@ import App from './containers/App';
 import Home from './containers/Home';
 
 export default (
-  <Route path="/" component={App}>
+  <Route path="/(:lang/)" component={App}>
```

Comment:
This should probably be `path="/(:lang/)firefox/"` but that could wait if you'd like.